.devops/deploy-pipelines.yml

```diff
@@ -157,3 +157,43 @@ stages:
               scriptLocation: inlineScript
               inlineScript: |
                 az webapp deploy --resource-group $(RESOURCE_GROUP) --name $(APP_NAME) --src-path ./target/api-config-$(pomversion.next).jar  --type jar
+
+      - job: integrationTests
+        dependsOn: deployJava
+        steps:
+          - script: |
+              yarn global add newman
+            displayName: 'newman installation'
+
+          - script: |
+              newman run api-test/ApiConfig.postman_collection.json --environment=api-test/Azure.postman_environment.json --reporters cli,junit --reporter-junit-export Results/api-config-TEST.xml
+            displayName: 'Run api test'
+            continueOnError: true
+
+          - task: PublishTestResults@2
+            inputs:
+              testResultsFormat: 'JUnit'
+              testResultsFiles: '**/*-TEST.xml'
+              searchFolder: '$(System.DefaultWorkingDirectory)'
+
+      - job: loadTests
+        dependsOn: deployJava
+        steps:
+          - script: |
+              yarn global add postman-to-k6
+            displayName: 'postman-to-k6 installation'
+          - script: |
+              yarn global add k6
+            displayName: 'k6 installation'
+
+          - script: |
+              postman-to-k6  api-test/ApiConfig.postman_collection.json -e api-test/Azure.postman_environment.json -o api-test/k6-script.js
+            displayName: 'Generate load test'
+            continueOnError: true
+
+          - script: |
+              k6 run --vus 10 --duration 5s api-test/k6-script.js
```

Comment:
we could verify some stuff like this https://github.com/Mattihew/k6-to-junit for k6 output report